MediaBrowser.Controller/Library/IUserManager.cs

```diff
@@ -154,17 +154,17 @@ public interface IUserManager
         /// <summary>
         /// Starts the forgot password process.
         /// </summary>
-        /// <param name="enteredUsername">The entered username.</param>
-        /// <param name="isInNetwork">if set to <c>true</c> [is in network].</param>
+        /// <param name="username">The entered username.</param>
         /// <returns>ForgotPasswordResult.</returns>
-        Task<ForgotPasswordResult> StartForgotPasswordProcess(string enteredUsername, bool isInNetwork);
+        Task<ForgotPasswordResult> StartForgotPasswordProcess(string username);
 
         /// <summary>
         /// Redeems the password reset pin.
         /// </summary>
-        /// <param name="pin">The pin.</param>
-        /// <returns><c>true</c> if XXXX, <c>false</c> otherwise.</returns>
-        Task<PinRedeemResult> RedeemPasswordResetPin(string pin);
+        /// <param name="code">The code.</param>
+        /// <param name="password">The password to apply if the code is correct.</param>
+        /// <returns>CodeRedeemResult.</returns>
+        Task<CodeRedeemResult> RedeemPasswordResetPin(string code, string password);
```

Comment:
Code seems like a too general variable. The method is called redeempasswordresetpin so the variable would be clearer if written as 'pincode' imo.